daq/forchestrator.py

```diff
@@ -87,9 +89,13 @@ def get_topology(self, path, params):
         """Get the network topology overview"""
         return self._collector.get_topology()
 
-    def get_active_host_route(self, path, params):
-        """Get active host route"""
-        return self._collector.get_active_host_route(params['src'], params['dst'])
+    def get_active_host_path(self, path, params):
+        """Get active host path"""
+        return self._collector.get_active_host_path(params['src'], params['dst'])
```

Comment:
pls add a TODO somewhere along the lines of "Rename self._collector to self._faucet_collector" and also about renaming faucet_states_collector to faucet_state_collector -- shouldn't change them in this PR, but just note it so we can address in the future.

Reply:
ACK

Reply:
Added above after declaration